src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to make new metrics available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with [Prometheus Operator](https://prometheus-operator.dev/) and a set of [Prometheus](https://prometheus.io/) shards in `agent mode`. These shards are monitoring agents that collect and forward critical cluster and workload metrics to a central [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
+
+No workload is the same, especially in the way it exposes its metrics, so the Observability Platform's monitoring configuration needs to be flexible. That's why it's based on the `ServiceMonitor` and `PodMonitor` Custom Resource Definitions (CRD) provided by the Prometheus Operator.
+
+Those allow you to:
+
+- Define where the metrics you want to ingest are (for example the container or port)
+- Transform metrics before ingesting them (for example dropping unneeded data, adding extra labels)
+
+You can learn more about the `ServiceMonitor` and `PodMonitor` CRD by checking the [Prometheus Operator API Docs](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md).
+
+## Prerequisites
+
+Before you start to ingest data from a running container, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics.
+
+Keep in mind that ingesting new metrics into the Observability Platform comes with a cost. The resource consumption of the central Mimir is related to the amount of metrics it has to handle. This means ingesting more metrics also leads to higher resource consumption of the Observability Platform overall.
+
+You can check the resource usage related to your ServiceMonitor and PodMonitor in the  `ServiceMonitors Overview` dashboard in your installation's Grafana.
+
+## Creating a ServiceMonitor
+
+Here is an example showing how to create a [ServiceMonitor](https://github.com/helm/charts/blob/master/stable/prometheus-operator/crds/crd-servicemonitor.yaml).
+This one targets a service named `my-service` in the `monitoring` namespace, and will route alerts to team `my-team`.  The manifests should be similar with any workload as long as you have a service that exposes the app's metrics.
+
+The bare minimum for a ServiceMonitor looks like this:
```

Comment:
I was just wondering...Should we describe how to use the CR (like "install" it)? Because I guess especially with out GitOps Setup this might confuse some folks. Do you put it into the observability folder of your MC, do you push it against the MC or against the WC and all that...I wouldn't know :D

Reply:
Well, maybe but I'm not sure how I would phrase it.

I think we should explain that the servicemonitor should be deployed as close as possible with the application so either via gitops, the helm chart and so on. So if you have some thoughts, I'm all ears

Reply:
Just waiting for ideas hère and then i'll merge :)